assets/Gruntfile.js

```diff
@@ -386,15 +386,24 @@ module.exports = function (grunt) {
         src: 'views/**/*.html',
         dest: '.tmp/scripts/templates.js',
         options: {
-          module: 'openshiftConsole',
+          module: 'openshiftConsoleTemplates',
+          standalone: true,
           htmlmin: '<%= htmlmin.dist.options %>',
-          usemin: 'scripts/scripts.js'
+          usemin: 'scripts/templates.js'
```

Comment:
Just thinking if you set `dest` to the `dist` directory and take out `usemin` do you need the extra copy task below?